run_page/generator/db.py

```diff
@@ -83,7 +83,9 @@ def update_or_create_activity(session, run_activity):
             if not location_country and start_point or location_country == "China":
                 try:
                     location_country = str(
-                        g.reverse(f"{start_point.lat}, {start_point.lon}")
+                        g.reverse(
+                            f"{start_point.lat}, {start_point.lon}", language="zh-CN"
```

Comment:
you may also do this in the `except` block below

Reply:
done